unstructured/ingest/connector/delta_table.py

```diff
@@ -0,0 +1,124 @@
+import logging
+import os
+import uuid
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Union
+
+import pandas
+import pyarrow
+from deltalake import DeltaTable
+from pyarrow import RecordBatch
+
+from unstructured.ingest.interfaces import (
+    BaseConnector,
+    BaseConnectorConfig,
+    BaseIngestDoc,
+    ConnectorCleanupMixin,
+    IngestDocCleanupMixin,
+    StandardConnectorConfig,
+)
+from unstructured.ingest.logger import make_default_logger
+from unstructured.utils import requires_dependencies
+
+
+@dataclass
+class SimpleDeltaTableConfig(BaseConnectorConfig):
+    verbose: bool
+    table_uri: Union[str, Path]
+    version: Optional[int] = None
+    storage_options: Optional[Dict[str, str]] = None
+    without_files: bool = False
+    columns: Optional[List[str]] = None
+    batch_size: Optional[int] = None
+    logger: Optional[logging.Logger] = None
```

Comment:
we should probably stick to only including primitives in the config in case this ever needs to be serialized